resources/public/api/conf/2.0/fhl_uk_property_business_annual_submission_create_and_amend.raml

```diff
@@ -0,0 +1,37 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.ruleTaxYearRangeInvalid
+  - errors.formatTaxYear
+  - errors.ruleTaxYearNotSupported
```

Comment:
This error message doesn't quite match up with this endpoint - the error message in the RAML states that this error is generated if the tax year specified is before the minimum tax year. That's true, but it doesn't mention that you might also get this error if the tax year exceeds the maximum tax year as well (as stated in business spec). It's a minor detail, up to you if you want to address this.